plugins/services/src/js/utils/HealthCheckUtil.js

```diff
@@ -2,6 +2,13 @@ import HealthCheckProtocols from '../constants/HealthCheckProtocols';
 
 module.exports = {
   isKnownProtocol(protocol) {
-    return ['', ...Object.values(HealthCheckProtocols)].includes(protocol);
+    return [
+      '',
+      ...[
```

Comment:
What's the purpose of spreading this array rather than just adding the three elements to the array explicitly?